integration_test/test_videoroom/test/integration/simulcast_test.exs

```diff
@@ -20,10 +16,44 @@ defmodule TestVideoroom.Integration.SimulcastTest do
   @set_peer_encoding_medium "simulcast-peer-medium-encoding"
   @simulcast_inbound_stats "simulcast-inbound-stats"
   @simulcast_outbound_stats "simulcast-outbound-stats"
-  @browser_options %{count: 1, delay: @peer_delay, headless: true}
-  @test_duration 240_000
-
-  @tag timeout: @test_duration
+  @browser_options %{count: 1, headless: true}
+  @max_test_duration 400_000
+
+  # we want to get stats for at least 30 seconds
+  # to ensure that the variant won't switch
+  # just after being selected;
+  # because gathering stats takes us 1 second
+  # (see get_stats in mustang.ex)
+  # we set stats_number to 15 and stats interval to 1s -
+  # we will fetch stats every 2 seconds
+  @stats_number 15
+  @stats_interval 1_000
+  # time needed to request and receive a variant
+  @variant_request_time 2_000
+  # max time needed to recognize variant as inactive
+  @variant_inactivity_time 2_000
+  # max time needed to recognize variant as active
+  @variant_activity_time 11_000
+  # times needed to probe from one resolution to another
+  # assumes that audio is present
+  # assumes the following max limits:
+  # audio - 50kbps
+  # video low - 150kbps
+  # video medium - 500kbps
+  # video high - 1500kbps
+  @probe_times %{low_to_medium: 15_000, low_to_high: 30_000, nil_to_high: 50_000}
```

Comment:
Were these values found empirically or is there a way to calculate them? I feel like it would be nice to include this information somewhere

Reply:
They were meassured empirically